docs/css/main.css

```diff
@@ -959,254 +981,6 @@ input[type=number] {
     text-align: center;
 }
 
-/* ============================================================= */

```

Comment:
I couldn't find where this classes are used and therefore removed it, so that I don't have to replace the colors here.

Reply:
Yeah, I also couldn't find it. I just left it earlier, but a good idea to clean up now though. I think earlier Penpa was written using Bootstrap JS package and so is that code, but then was moved to vanilla JS.